management/src/main/java/io/micronaut/management/endpoint/EndpointSensitivityProcessor.java

```diff
@@ -71,8 +71,7 @@ public void process(BeanDefinition<?> beanDefinition, ExecutableMethod<?, ?> met
             boolean sensitive = configuration
                 .isSensitive()
                 .orElseGet(() -> beanDefinition
-                    .getValue(Endpoint.class, "defaultSensitive", Boolean.class)
-                    .orElse(Endpoint.SENSITIVE));
+                    .isTrue(Endpoint.class, "defaultSensitive"));
```

Comment:
This method is automatically retrieving the default value if the value isn't specified?

Reply:
No it isn't, but in this case the default is `false` hence why it made sense for this use case, and the performance is better.